PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -3626,6 +3626,7 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 
 		case NPPM_INTERNAL_SETTING_TAB_REPLCESPACE:
 		case NPPM_INTERNAL_SETTING_TAB_SIZE:
+		case NPPM_INTERNAL_SETTING_BACKSPACE_UNINDENTS:

```

Comment:
Please change the following:
1. remove the definition of `NPPM_INTERNAL_SETTING_BACKSPACE_UNINDENTS` and comment the drfinition of `NPPM_INTERNAL_SETTING_TAB_SIZE`.
2. rename `NPPM_INTERNAL_SETTING_TAB_REPLCESPACE` to `NPPM_INTERNAL_SET_TAB_SETTINGS`.